native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mssql-native/reflect-config.json

```diff
@@ -0,0 +1,103 @@
+[
+  {
+    "name": "com.microsoft.sqlserver.jdbc.SQLServerDataSource",
+    "queryAllPublicMethods": true,
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": [ ]
+      },
+      {
+        "name": "setEncrypt",
+        "parameterTypes": [
+          "boolean"
+        ]
+      },
+      {
+        "name": "setKeyStoreAuthentication",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setKeyStoreLocation",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setKeyStoreSecret",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setLoginTimeout",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setPassword",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setQueryTimeout",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setSocketTimeout",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setTrustServerCertificate",
+        "parameterTypes": [
+          "boolean"
+        ]
+      },
+      {
+        "name": "setTrustStore",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setTrustStorePassword",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setURL",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setUser",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "com.zaxxer.hikari.HikariConfig",
```

Comment:
I guess this is a dependency of sql. We have add this class to reflection in sql. Does removing this config gives an error?

Reply:
Yes, error:
```error: Error in SQL connector configuration: Property ConnectionTimeout does not exist on target class com.zaxxer.hikari.HikariConfig```

Reply:
`com.zaxxer.hikari` is not a dependency added by mssql right?
In that case we need to remove this here and add this to sql. We might have missed this use case in sql tests. Any idea on where we call this `ConnectionTimeout`?

Reply:
Updated in SQL  https://github.com/ballerina-platform/module-ballerina-sql/pull/569

Reply:
Lets update the new sql version : `1.5.1-20221013-124900-90866bf` and test